### DISCUSSION TITLE
Min + max block types in matrix fields

### DISCUSSION THREAD

**Discussion by josh-attwood** [craftcms/cms#12197] [Ideas] [answered]
It would be great to have some function to limit the number of times a block type can be added to a matrix field. 

For example we have a site where we'd like a certain block to always be shown on the homepage but would give the user the ability to choose its position and would like to make sure it only shows up once. In that scenario we'd set the min and max to 1. This is something I've used in the past when developing in Wordpress + ACF which was very useful.

**Comment by brandonkelly** [comment, marked as answer]
Thanks for posting! This was previously requested over at #8596. You can follow along over there for updates.
